### PR TITLE
KMA-8 Set up tests, add tests for common/button

### PR DESCRIPTION
**Pull request by ericgmoon**

### CODE REVIEW

File: src/packages/common/button/styles.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
     paddingTop: 16,
     paddingBottom: 16,
+    paddingLeft: 12,
+    paddingRight: 12,
     background: "#000",
   },
-  icon: {
-    paddingLeft: 16,
+  buttonIconRight: {
+    paddingLeft: 20,
```
Comment (nzh20) [inline src/packages/common/button/styles.js:11]: If we gonna have consistent padding sizes - we could make a package for constants

Comment (ericgmoon) [reply src/packages/common/button/styles.js:11]: That is a really good idea - creating a ticket for it on our board now. Maybe that's better as a separate PR, as @GeorgeKandamkolathy has been working on another component himself.

**Review by nzh20 — APPROVED** [review]
Looks fine - didn't rlly look into some of the config changes just assumed you did it right.

Possibly consider shared constants package later

**Comment by ericgmoon** [comment]
As requested by @nzh20, here is a screenshot of the renders:
image

**Pull request merged**